rpcs3/Gui/GameViewer.h

```diff
@@ -88,12 +88,12 @@ struct ColumnsArr
 		m_img_list = new wxImageList(80, 44);
 
 		m_columns.clear();
-		m_columns.emplace_back(m_columns.size(),  90, "Icon");
+		m_columns.emplace_back(m_columns.size(),  90, "");
 		m_columns.emplace_back(m_columns.size(), 160, "Name");
 		m_columns.emplace_back(m_columns.size(),  85, "Serial");
-		m_columns.emplace_back(m_columns.size(),  55, "FW");
-		m_columns.emplace_back(m_columns.size(),  55, "App version");
-		m_columns.emplace_back(m_columns.size(),  75, "Category");
+		m_columns.emplace_back(m_columns.size(),  55, "Min fw");
```

Comment:
Nitpicking here, but could you put a dot here, after the "Min", since it's a shortened word. Like so: "Min. FW"
Same for other shortened words, would be nice.